backend/database/factories/ProductRevisionFactory.php

```
@@ -0,0 +1,41 @@
+        return [
+            'product_type_id' => ProductType::factory(),
+            'revision_code' => $codes[$counter++ % count($codes)].fake()->unique()->numberBetween(1, 99999),
+            'description' => fake()->optional()->sentence(),
+            'lifecycle_status' => RevisionLifecycle::Draft,
+            'process_template_id' => null,
+        ];
+    }
+
+    public function released(): static
+    {
+        return $this->state(fn (array $attributes) => [
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$attributes'. (undefined)

(UnusedFormalParameter)